src/nwb_datajoint/spikesorting/spikesorting_curation.py

```diff
@@ -607,7 +636,7 @@ class FinalizedSpikeSortingSelection(dj.Manual):
 
 
 @schema
-class FinalizedSpikeSorting(dj.Manual):
+class FinalizedSpikeSorting(dj.Computed):
```

Comment:
I would suggest calling this `CuratedSpikeSorting` - I think it is more descriptive than `FinalizedSpikeSorting`.

Reply:
Sounds good.